database/events.api.php

```diff
@@ -106,4 +106,100 @@ function delete_event($event_id)
 function publish_event($event_id, $state)
 {
     return query_db("UPDATE deplacements SET open=? WHERE did=? LIMIT 1", $state, $event_id);
+}
+
+function save_registration($event_id, $user_id, $post, $competitions_id)
```

Comment:
This does not belong in this PR. Please create a separate PR or remove this change.